cmd/node/node.go

```diff
@@ -205,15 +208,48 @@ func LoadConfigFromFile() (*cfg.Config, error) {
 	}
 
 	conf := cfg.DefaultConfig()
+
+	hook := mapstructure.ComposeDecodeHookFunc(
+		mapstructure.StringToTimeDurationHookFunc(),
+		mapstructure.StringToSliceHookFunc(","),
+		bigRatDecodeFunc(),
+	)
+
 	// load config if it was loaded to our viper
-	err := vip.Unmarshal(&conf)
+	err := vip.Unmarshal(&conf, viper.DecodeHook(hook))
 	if err != nil {
-		log.Error("Failed to parse config", log.Err(err))
+		log.With().Error("Failed to parse config", log.Err(err))
 		return nil, err
 	}
 	return &conf, nil
 }
 
+func bigRatDecodeFunc() mapstructure.DecodeHookFunc {
+	return func(f reflect.Type, t reflect.Type, data interface{}) (interface{}, error) {
+		if t != reflect.TypeOf(&big.Rat{}) {
+			return data, nil
+		}
+
+		switch f.Kind() {
+		case reflect.String:
+			v, ok := new(big.Rat).SetString(data.(string))
+			if !ok {
+				return nil, errors.New("malformed string representing big.Rat was provided")
+			}
+
+			return v, nil
+		case reflect.Float64:
```

Comment:
`f` can't be smaller type?

Reply:
I think it will come from `json`s/`yaml`s/`toml`s as `float64`, so it's enough. Actually, I need string only. Do you think I should add all numeric types?

Reply:
probably unnecessary